sdk/objectanchors/Azure.MixedReality.ObjectAnchors/README.md

```diff
@@ -0,0 +1,144 @@
+```
+
+From .NET CLI
+
+```dotnetcli
+dotnet add package Azure.MixedReality.ObjectAnchors
+```
+
+Add a package reference:
+
+```xml
+<PackageReference Include="Azure.MixedReality.ObjectAnchors" Version="0.1.0-beta.0" />
+```
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://azure.microsoft.com/free/).
+- You must have an [Azure Object Anchors](https://review.docs.microsoft.com/azure/object-anchors/) account.
+
+### Authenticate the client
+
+Mixed Reality services support a few different forms of authentication:
+
+- Account Key authentication
+  - Account keys enable you to get started quickly with using Mixed Reality services. But before you deploy your application
+    to production, we recommend that you update your app to use Azure AD authentication.
+- Azure Active Directory (AD) token authentication
+  - If you're building an enterprise application and your company is using Azure AD as its identity system, you can use
+    user-based Azure AD authentication in your app. You then grant access to your Object Anchors account by using your
+    existing Azure AD security groups. You can also grant access directly to users in your organization.
+  - Otherwise, we recommend that you obtain Azure AD tokens from a web service that supports your app. We recommend this
+    method for production applications because it allows you to avoid embedding the credentials for access to the Object Anchors asset conversion service in your client application.
+
+See [here](https://docs.microsoft.com/azure/spatial-anchors/concepts/authentication) for detailed instructions and information.
+
+## Key concepts
+
+### ObjectAnchorsClient
+
+The `ObjectAnchorsClient` is the client library used to access the Object Anchors asset conversion service. From there, a storage upload URI will be provided for users to upload their assets for conversion into a format that is usable by the Object Anchors runtime.
+
+Assets uploaded to the Microsoft-hosted URI obtained from the client will be retained for **48 hours**.
+
+The final converted model in Microsoft-hosted storage will be retained for **48 hours**. 
+
+## Examples
+
+### Upload an asset for Object Anchors asset conversion
+
+```csharp
+TokenCredential credential = new DefaultAzureCredential(true);
+
+ObjectAnchorsClient client = new ObjectAnchorsClient(account, credential);
+
+Uri uploadedInputAssetUri = (await client.GetAssetUploadUriAsync()).Value.InputAssetUploadUri;
+
+BlobClient blobClient = new BlobClient(uploadedInputAssetUri);
+
+using (FileStream fs = File.OpenRead(localFilePath))
+{
+  await blobClient.UploadAsync(fs);
+}
+```
+
+### Begin Object Anchors asset conversion after uploading an asset
+
+```csharp
+StartAssetConversionOptions ingestionJobOptions = new StartAssetConversionOptions(uploadedInputAssetUri, AssetFileType.FromFilePath(localFilePath), assetGravity, scale);
+
+AssetConversionOperation operation = await client.StartAssetConversionAsync(ingestionJobOptions);
```

Comment:
```suggestion
StartAssetConversionOptions assetConversionOptions = new StartAssetConversionOptions(uploadedInputAssetUri, AssetFileType.FromFilePath(localFilePath), assetGravity, scale);

AssetConversionOperation operation = await client.StartAssetConversionAsync(assetConversionOptions);
```

The term "ingestion" doesn't mean much to customers. I'd go with either "assetConversion" or "conversion".

Reply:
Pending

Reply:
Ingestion replaced with assetconversion